src/consensus/aft/raft.h

```diff
@@ -176,6 +184,22 @@ namespace aft
       return leader_id;
     }
 
+    std::set<NodeId> backups()
```

Comment:
I'm a little worried about this, because this is really the latest backups, but not necessarily all the backups a given Tx ought to go to, for example if a node has been removed but is behind. Am I missing something?

Reply:
I think this should all_active_backups() and be the union of all backups in active configurations.